lib/Serialization/SerializedModuleLoader.cpp

```diff
@@ -565,17 +565,20 @@ SerializedModuleLoaderBase::findModule(AccessPathElem moduleID,
         case SearchPathKind::RuntimeLibrary: {
           isFramework = false;
 
-          bool checkTargetSpecificModule;
+          // Apple platforms always use target-specific files within a
+          // .swiftmodule directory for the stdlib; non-Apple platforms always
+          // use single-architecture swiftmodules.
```

Comment:
I think the "non-Apple platforms" part of this comment is now out of date. (But I'm going to suggest rewriting it below.)